examples/encrypting.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::Result;
+use ironoxide::prelude::*;
+use std::{fs::File, path::PathBuf};
+
+/// This is a very basic example of encrypting to a user_id and a group_id.
+/// The group that's encrypted to is created each run, but the user is just the current user.
+#[tokio::main]
+async fn main() -> Result<()> {
+    let (device_context, sdk) =
+        initialize_sdk_from_file(&PathBuf::from("examples/example-ironoxide-user.json")).await?;
+    encrypt_to_group(&sdk).await?;
+    encrypt_to_user(&sdk, device_context.account_id()).await?;
+    Ok(())
+}
+///Create a random group and encrypt the super secret message to it.
```

Comment:
In the most nit-picky way possible, a space after the `///` would make this more consistent.